src/Kentico.Xperience.Lucene/Admin/UIPages/IndexListing.cs

```diff
@@ -212,10 +212,10 @@ private Row GetRow(LuceneIndexStatisticsViewModel statistics)
                     //{
                     //    Value = statistics.LastBuildTimes.ToString()
                     //},
-                    new StringCell
-                    {
-                        Value = statistics.UpdatedAt.ToString()
-                    },
+                    //new StringCell
```

Comment:
If we're not using this, can it be removed? We have the git history if we need to recover it.